nautobot/docs/configuration/optional-settings.md

```diff
@@ -478,16 +480,18 @@ Please see the [official Django documentation on `FORCE_SCRIPT_NAME`](https://do
 
 Default: `os.path.join(NAUTOBOT_ROOT, "git")`
 
-The file path to a directory where cloned [Git repositories](../models/extras/gitrepository.md) will be located.
+Environment Variable: `NAUTOBOT_GIT_ROOT`
 
-The value of this variable can also be customized by setting the environment variable `NAUTOBOT_GIT_ROOT` to a directory path of your choosing.
+The file path to a directory where cloned [Git repositories](../models/extras/gitrepository.md) will be located.
 
 ---
 
 ## GIT_SSL_NO_VERIFY
 
 Default: Unset
 
+Environment Variable: `GIT_SSL_NO_VERIFY`
+
 If you are using a self-signed git repository, you will need to set the environment variable `GIT_SSL_NO_VERIFY="1"`
 in order for the repository to sync.
```

Comment:
This one's a bit confusing -- *only* the environment variable `$GIT_SSL_NO_VERIFY` is relevant. Configuring `settings.GIT_SSL_NO_VERIFY` would do nothing. I feel like adding a separate `Environment Variable:` entry here is actually a bit more confusing versus just reading the textual description, personally.